packages/flutter/test/material/ink_sparkle_test.dart

```diff
@@ -48,16 +48,15 @@ void main() {
     final Finder buttonFinder = find.text('Sparkle!');
     await tester.tap(buttonFinder);
     await tester.pump();
-    await tester.pumpAndSettle();
```

Comment:
Im not really sure why this worked before. I think pumpAndSettling past the animation duration still worked since we would continue painting it forever